dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -42,5 +42,5 @@ properties:
       or for K_PRIO_PREEMPT(2)
         priority = <2 PRIO_PREEMPT>;
 
-      When this property is missing a default priority of <0 PRIO_COOP> is
+      When this property is missing a default priority of <0 PRIO_PREEMPT> is
```

Comment:
Regarding https://github.com/zephyrproject-rtos/zephyr/commit/7f51907fda2dd30f18d01262f06c175714a9be3f
Should "zephyr,priority" property become required?

Reply:
IMO that we should avoid that as much as possible. As this issue is proving setting a proper value for `zephyr,priority` requires a bit of knowledge of the zephyr internals and that could be a bit too much for the average user. I very much prefer leaving a reasonable value as default that a more advanced user can tweak if they know what they are doing.